cclient/response/attach_to_tangle.c

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2018 IOTA Stiftung
+ * https://github.com/iotaledger/entangled
+ *
+ * Refer to the LICENSE file for licensing information
+ */
+
+#include "attach_to_tangle.h"
+
+attach_to_tangle_res_t* attach_to_tangle_res_new() {
+  attach_to_tangle_res_t* res = NULL;
+  utarray_new(res, &ut_str_icd);
+  return res;
+}
+
+const char* attach_to_tangle_res_trytes_at(attach_to_tangle_res_t* res,
+                                           int index) {
+  if (utarray_len(res) > index) {
+    char** p = NULL;
+    p = (char**)utarray_eltptr(res, index);
+    if (p) {
+      return *p;
+    }
+  }
+  return '\0';
```

Comment:
Shouldn't this return `NULL` ? It's expected to return a `char *`, you're returning a `char`.